src/pudl/package_data/eia860/column_maps/boiler_info.csv

```diff
@@ -0,0 +1,39 @@
+operating_year,inservice_yr,inservice_yr,inservice_yr,inservice_year,inservice_year,inservice_year,inservice_year,inservice_year,inservice_year,inservice_year,inservice_year,inservice_year
+retirement_month,retirement_mm,retirement_mm,retirement_mm,retirement_month,retirement_month,retirement_month,retirement_month,retirement_month,retirement_month,retirement_month,retirement_month,retirement_month
+retirement_year,retirement_yr,retirement_yr,retirement_yr,retirement_year,retirement_year,retirement_year,retirement_year,retirement_year,retirement_year,retirement_year,retirement_year,retirement_year
+firing_type_1,,,,,firing_type_1,firing_type_1,firing_type_1,firing_type_1,firing_type_1,firing_type_1,firing_type_1,firing_type_1
+firing_type_2,,,,,firing_type_2,firing_type_2,firing_type_2,firing_type_2,firing_type_2,firing_type_2,firing_type_2,firing_type_2
+firing_type_3,,,,,firing_type_3,firing_type_3,firing_type_3,firing_type_3,firing_type_3,firing_type_3,firing_type_3,firing_type_3
+max_steam_flow_thousand_pounds_per_hour,max_steam_flow,max_steam_flow,max_steam_flow,max_steam_flow_thousand_pounds_per_hour,max_steam_flow_thousand_pounds_per_hour,max_steam_flow_thousand_pounds_per_hour,max_steam_flow_thousand_pounds_per_hour,max_steam_flow_thousand_pounds_per_hour,max_steam_flow_thousand_pounds_per_hour,max_steam_flow_thousand_pounds_per_hour,max_steam_flow_thousand_pounds_per_hour,max_steam_flow_thousand_pounds_per_hour
+firing_rate_using_coal_0_1_tons_per_hour,coal_fire_steam_flow,coal_fire_steam_flow,coal_fire_steam_flow,coal_fire_steam_flow_0_1_tons_per_hour,firing_rate_using_coal_0_1_tons_per_hour,firing_rate_using_coal_0_1_tons_per_hour,firing_rate_using_coal_0_1_tons_per_hour,firing_rate_using_coal_0_1_tons_per_hour,firing_rate_using_coal_0_1_tons_per_hour,firing_rate_using_coal_0_1_tons_per_hour,firing_rate_using_coal_0_1_tons_per_hour,firing_rate_using_coal_0_1_tons_per_hour
+firing_rate_using_petroleum_0_1_barrels_per_hour,pet_fire_steam_flow,pet_fire_steam_flow,pet_fire_steam_flow,petroleum_fire_steam_flow_0_1_barresl_per_hour,firing_rate_using_petroleum_0_1_barrels_per_hour,firing_rate_using_petroleum_0_1_barrels_per_hour,firing_rate_using_petroleum_0_1_barrels_per_hour,firing_rate_using_petroleum_0_1_barrels_per_hour,firing_rate_using_petroleum_0_1_barrels_per_hour,firing_rate_using_petroleum_0_1_barrels_per_hour,firing_rate_using_petroleum_0_1_barrels_per_hour,firing_rate_using_petroleum_0_1_barrels_per_hour
+firing_rate_using_gas_0_1_mcf_per_hour,,,,,firing_rate_using_gas_0_1_mcf_per_hour,firing_rate_using_gas_0_1_mcf_per_hour,firing_rate_using_gas_0_1_mcf_per_hour,firing_rate_using_gas_0_1_mcf_per_hour,firing_rate_using_gas_0_1_mcf_per_hour,firing_rate_using_gas_0_1_mcf_per_hour,firing_rate_using_gas_0_1_mcf_per_hour,firing_rate_using_gas_0_1_mcf_per_hour
+firing_rate_using_gas_0_1_tons_per_hour,gas_fire_steam_flow,gas_fire_steam_flow,gas_fire_steam_flow,gas_fire_steam_flow_0_1_tons_per_hour,,,,,,,,
```

Comment:
What does the `0_1` mean in these column names?

Reply:
It means 0.1, which is itself odd. The field in 860 is 'Firing Rate Using Coal (0.1 Tons per Hour)' and the description is 'Design firing rate at maximum continuous steam flow for coal (nearest 0.1 ton per hour)'. I'm not sure how best to represent this, though `0_1` is probably not the right way.

Reply:
Ah okay. Do you think they're ever really describing the units (tenths of a ton per hour) or do you think it's always describing the precision of the values, and they just messed up the annotation? If it's just describing the precision, then I think that's information that the filers need, but we don't need to preserve it that part of the column names, since we're just going to take whatever the reported values are (which will hopefully reflect that stated precision).  In which case we'd probably go with something like:

* `firing_rate_coal_tons_per_hour`
* `firing_rate_oil_bbls_per_hour`
* `firing_rate_gas_mcf_per_hour`

The last one `firing_rate_using_gas_0_1_tons_per_hour` seems a little fishy. Is it really a firing rate for gas? It looks more like a steam flow rate in tons of water/steam. Can't tell if that's an input rate (e.g. into an HRSG) or an output rate (quantity of steam generated by a boiler).

Reply:
Agreed on that the `0_1` represents precision, a point I was missing while I was looking at this before. Will remove the numbers as you suggest.

`firing_rate_using_gas_0_1_tons_per_hour` is a mistake in the column name, I checked the Layout file and the values across years it is in mcf per hour like the other years. So now that they are all in units that make sense for fuel flow rate, my interpretation is that this is rate of fuel consumption for maximum continuous steam flow.